text.cabal

```diff
@@ -93,11 +93,17 @@ flag pure-haskell
   default: False
   manual: True
 
+flag repl-tests
+  description: Enable running tests in the repl
```

Comment:
@chreekat How do you invoke tests in the repl? Sorry, I have not used `ghcid` for ages and failed to figure out myself.

Reply:
Aha, I suppose it's something like `ghcid --command='cabal v2-repl test:tests' --test 'Main.main'`.

Reply:
I've just published `tasty-inspection-testing-0.2.1` which detects if it is executed inside ghci and skips inspections tests automatically. Could you please `cabal update` and check the flag is still needed?

Reply:
Yes, that did work, but to be honest I was also using the flag to run the tests with --disable-optimisation. :) 

That being the case, the flag has a bad name.

I'll just remove it from this PR so we can think about it more in a different context.